README.md

```
@@ -412,18 +455,20 @@ pnpm mcp --emails=your.email@fwc.gov.au
 mcp-inspector pnpm mcp --emails=your.email@fwc.gov.au
 ```
 
-### CLI Mode Issues
+### Authentication Issues
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)